python/ql/test/experimental/query-tests/Security/CWE-287-ConstantSecretKey/app_safe.py

```diff
@@ -0,0 +1,22 @@
+from flask import Flask, session
+from flask_session import Session
```

Comment:
Can you highlight for me how `flask_session` makes this safe? (preferably in a code comment so it doesn't get lost)